### PR TITLE
Updated specs to version 1.3 beta

### PR DESCRIPTION
**Pull request by olivier-kasisto**
Added more details on the rules 
Simplify the specs for the service bank_location
Add new features on accounts
Add new service to submit customer actions (Activate credit card / Fee waiver)
Cleanup the objects

### CODE REVIEW

File: api-overview.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Version 1.2.1
   * [/payees](#payees)
 - [Bank Locations Methods](#bank-locations-methods)
   * [/bank_locations](#bank-locations)
+- [Customer Service Request Methods](#customer-service-request-methods)
+  * [/customer_service_request](#submit-customer-service-request)
```
Comment (bstewartny) [inline api-overview.md]: I suggest renaming to "Customer Action Methods" and "/customer_action"

File: api-overview.md [hunk 2]
```diff
@@ -476,7 +528,17 @@ token: string (optional)
             "value": "string"
         }
     ],
-    "location": "string",
+    "location": {
+        "address": "string",
+        "city": "string",
+        "state": "string", 
+        "zip": "string",
+        ""
```
Comment (bstewartny) [inline api-overview.md]: where is country?

File: api-overview.md [hunk 3]
```diff
@@ -865,58 +928,127 @@ token: string (optional)
...
-            "day": "string",
-            "hours": { "start": 0, "end" : 0}
-        }
-    ],
-    "lobby_hours": [
-        {
-            "day": "string",
-            "hours": { "start": 0, "end" : 0}
+    "opening_hours": { // I propose to use the below structure as it makes it easier for us to manage localized schedule. And we don't support queries like "Sow me the branches open today"
+        "mon - fri": "9:00am - 6:00pm",
```
Comment (bstewartny) [inline api-overview.md]: I think we need one value for each day: ["mon":{"open":"9:00am", "close":"6:00pm"}, "tue":{...}, "wed":{...},...]
We probably need more structure around special dates (holidays) which have different hours.

File: api-overview.md [hunk 4]
```diff
@@ -865,58 +928,127 @@ token: string (optional)
...
+    }
+}]
+```
+
+### Customer Service Request Methods
+
+#### Submit Customer Service Request
+
+```
+POST /customer_service_request
```
Comment (bstewartny) [inline api-overview.md]: Rename "/customer_action"

File: api-overview.md [hunk 5]
```diff
@@ -865,58 +928,127 @@ token: string (optional)
...
+Accept: application/json
+secret: string
+token: string
+locale: string
+request_id: string
+Date: Tue, 01 Jan 2017 00:00:00 GMT
+```
+```json
+{
+    "request_type": "credit_card_activate",
```
Comment (bstewartny) [inline api-overview.md]: Rename "request_type" to "action_name"

File: api-overview.md (hunk 3 above)
Comment (davidsawicki) [reply api-overview.md]: @olivier-kasisto , I am with @bstewartny on this one. In fact we *can* parse dates, and we do; we do it for Wells Fargo currently

File: api-overview.md [hunk 6]
```diff
@@ -324,10 +343,43 @@ token: string (optional)
...
+| payment_due_amount |  |  | x | x | x |  |
+| payment_due_date |  |  | x | x | x |  |
+| minimum_payment_due_amount |  |  | x | x | x |  |
+| reward_points |  | x | x |  |  | x |
+| reward_miles |  | x | x |  |  | x |
+| reward_cashback |  | x | x |  |  | x |
+| can_transfer_to | x | x | x | x | x | x |
+| can_transfer_from |  | x | x |  |  | x |
+| can_pay_payee |  | x | x |  |  | x |
+| can_waive_fee | x | x | x | x | x | x |
```
Comment (davidsawicki) [inline api-overview.md]: @olivier-kasisto why do you put this table here? This is in fact the point of having the reasoner table, which we require from our customers. They should tell us what accounts support which features, not us...

Comment (olivier-kasisto) [reply api-overview.md]: @davidsawicki When we go to a new Bank and ask them for the mapping, we don't have any supporting document and it's very difficult to explain to them how to do this exercise.
I added this table to try to make the exercise easier.

**Pull request merged**